sdks/java/core/src/main/java/org/apache/beam/sdk/io/LocalFileSystem.java

```diff
@@ -46,6 +46,7 @@
 import org.apache.beam.sdk.io.fs.MatchResult;
```

Comment:
We should add a comment about paths that are expected to work across mac, linux and windows as examples above so people know how to specify them.